test/source/test.ts

```diff
@@ -199,6 +203,8 @@ test.after.always('send debug info if any', async t => {
   const debugHtmlAttachments = getDebugHtmlAtts(testId, mockApiLogs);
   if (debugHtmlAttachments.length) {
     console.info(`FAIL ID ${testId}`);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
let's revert `mockApi` back to optional property to not use `@ts-ignore`

Reply:
> Ok, how about we disable no-non-null-assertion rule like @tomholub suggested? Adding // eslint-disable-next-line @typescript-eslint/no-non-null-assertion to every t.mockApi.attesterConfig =  doesn't seem like good idea too.

Reply:
Ok, let's use this solution for now